feature/experimental/p4rt/ate_tests/base_p4rt/base_p4rt_test.go

```diff
@@ -285,11 +284,31 @@ func verifyReadReceiveMatch(expected_update []*p4_v1.Update, received_entry *p4_
 
 	matches := 0
 	for _, table := range received_entry.Entities {
-		if diff := cmp.Diff(expected_update[0].Entity.Entity, table.Entity, protocmp.Transform()); diff != "" {
-			glog.Errorf("Table entry diff (-want +got): \n%s", diff)
-			continue
+		//add a switch case for the captured table entry
+		//Switch case to remove the counter_data and meter_config from table entry returned by p4rt server
+		var tableMap map[string]interface{}
+		switch entry := table.Entity.(type) {
+		case *p4_v1.Entity_TableEntry:
+			ent1, _ := json.Marshal(entry)
+			var toMap map[string]interface{}
+			_ = json.Unmarshal([]byte(string(ent1)), &toMap)
+			tableMap = toMap["TableEntry"].(map[string]interface{})
+			delete(tableMap, "meter_config")
+			delete(tableMap, "counter_data")
+		default:
+			fmt.Println("Not a table entry")
 		}
-		matches++
+		ent2, _ := json.Marshal(expected_update[0].Entity.Entity)
```

Comment:
why ignore the error in the Marshal and Unmarshal calls?

Reply:
Hi Greg, 

I will add a variable for the error there in the next commit. Thanks